src/client/game.cpp

```diff
@@ -4299,6 +4306,10 @@ void Game::readSettings()
 	m_cache_cam_smoothing = rangelim(m_cache_cam_smoothing, 0.01f, 1.0f);
 	m_cache_mouse_sensitivity = rangelim(m_cache_mouse_sensitivity, 0.001, 100.0);
 
+	m_invert_mouse = g_settings->getBool("invert_mouse");
+	m_enable_hotbar_mouse_wheel = g_settings->getBool("enable_hotbar_mouse_wheel");
+	m_invert_hotbar_mouse_wheel = g_settings->getBool("invert_hotbar_mouse_wheel");
```

Comment:
Forgot to mention this, but you also have to do a `registerChangedCallback` call for each setting in `Game()`, and also deregister again in `~Game()`.

(While you're at it, you could also add the missing listeners for the other settings that are read here, e.g. `pause_on_lost_focus`.)

Reply:
Done.